src/screens/PageHeader.js

```diff
@@ -0,0 +1,281 @@
+import React from 'react';
+import { Anchor, Button, PageHeader } from 'grommet';
+import Page from '../components/Page';
+import { genericSyntaxes } from '../utils/props';
+import Item from './Components/Item';
+import {
+  GenericA11yTitle,
+  GenericAlignSelf,
+  GenericGridArea,
+  GenericMargin,
+} from '../utils/genericPropExamples';
+import {
+  ComponentDoc,
+  Properties,
+  Property,
+  PropertyValue,
+  Description,
+  Example,
+  ThemeDoc,
+} from '../components/Doc';
+
+const extendedGenericSyntaxes = JSON.parse(JSON.stringify(genericSyntaxes));
+extendedGenericSyntaxes.margin.unshift('none');
+
+const PageHeaderPage = () => (
+  <Page>
+    <ComponentDoc
+      name="PageHeader"
+      availableAt={[
+        {
+          url: 'https://storybook.grommet.io/?selectedKind=Type-Paragraph&full=0&stories=1&panelRight=0',
+          badge:
+            'https://cdn-images-1.medium.com/fit/c/120/120/1*TD1P0HtIH9zF0UEH28zYtw.png',
```

Comment:
This is taking us to the Paragraph stories instead of PageHeader

Reply:
Fixed.